app/services/symphony_client.rb

```diff
@@ -76,7 +76,7 @@ def renew_items(user, checkouts)
 
       case response.status
       when 200
-        status[:success] << checkout
+        status[:success] << { renewal: checkout, error_message: nil }
```

Comment:
what do you think for using just `message` instead of `error_message` for both, so a bit more generic?
not so important though, just a symbol

Reply:
I like it, much better. That was what I wrote in Slack initially 😉

Reply:
error_message is also taken in that it's a method, I was just kinda going off what you had put in the other branch. I'll update and push.

Reply:
updated in latest, used `sirsi_response` 🤷‍♂

Reply:
I liked sirsi_response better 👍